org.gnome.Evince.json

```diff
@@ -53,7 +53,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/uclouvain/openjpeg.git",
-                    "commit": "d2205ba2ee78faeea659263383446c4472b1f9df"
```

Comment:
openjpg is avalaible in gnome runtime. I think you can remove this module completely

Reply:
My local build succeeded without the module, I'll remove it.

Reply:
This should be accompanied with a `-DENABLE_LIBOPENJPEG=...` in the poppler module's configure options, to make sure that JPEG2000 images support isn't silently disabled.